zipline/_protocol.pyx

```diff
@@ -153,6 +153,9 @@ cdef class BarData:
     data_frequency : {'minute', 'daily'}
         The frequency of the bar data; i.e. whether the data is
         daily or minute bars
+    rl_controller : callable
```

Comment:
`callable` isn't the right annotation here.  We just just put `Restrictions` (probably with the module path when we decide on it.)